devlog/_plan/260820_bug_pr_backlog_consolidation/010_layer1_bearer_admission_2132.md

```
@@ -0,0 +1,70 @@
+> **SUPERSEDED IN PART — read `000_research_inventory.md` amendments 1 and 2 first.**
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)